resources/case_studies/Insurance/InsuranceClient.obs

```diff
@@ -2,9 +2,27 @@ import "resources/case_studies/Insurance/InsuranceService.obs"
 import "Obsidian Runtime/src/Runtime/edu/cmu/cs/obsidian/java-utilities/System.obs"
 
 main contract InsuranceClient {
-	transaction main(remote InsuranceService i) {
-         Bid b = i.requestBids(3);
-         System.print("bid : 3");
+	transaction main(remote InsuranceService is) {
```

Comment:
"is" is a bit too succinct, since it is an English word with a different meaning.